packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -51,11 +60,13 @@ const ItemDisplay = ({
   isTruncated,
   isDisabled,
   icon,
+  iconAlignment,
   ...customProps
 }) => {
   const componentClassNames = cx([
     'item-display',
     { 'is-disabled': isDisabled },
+    { [`icon-${iconAlignment}`]: iconAlignment && icon },
```

Comment:
Since `iconAlignment` defaults to `center` (truthy) if not provided, ```[`icon-${iconAlignment}`]: icon``` should suffice.

Reply:
Considered that, definitely is truthy - didn't know whether to be more explicit/protectionist, or to be more liberal.

Reply:
Applied https://github.com/cerner/terra-clinical/pull/490/commits/64a30bf5d3762a293a49c28184defc2bfc287654